### ISSUE TITLE
Changing street names

### ISSUE THREAD

**Issue by BrackoNe** [IpswichMapper/SwiftAddress#13]
This one is enhancement.

You have implemented a great feature to grab a street name by reverse geocoding and to keep the surveyor informed by showing different color schemes for different street names. Really great!

Anyway, when the building is far away, it sometimes snaps to the wrong street, so my proposals are:
* Grab all streets near you at the moment of activating the "Modify Street Name" button. Display all the street names within 50 meters (for example) as a list. Please also offer the manual input since there might be buildings that don't belong to any of the streets (these are in their own streets/squares, etc.), or the street is not mapped in OSM yet.
* Why the changing the street isn't straightforward? Isn't that easier to tap on the top right label on the street name rather than swiping the "0" button?

**Comment by IpswichMapper** [comment]
Done. 1f8d0cc

I chose to use a different solution to what you suggested. I still kept the textbox, but instead I used a type of textbox called "autocompletetextview". This automatically starts showing suggestions when you are typing in the street name:

![Screenshot_20210222-112232_SwiftAddress](https://user-images.githubusercontent.com/72318451/108702429-291b3c80-7501-11eb-9cd5-e3ad80ae1ea8.png)
